src/transform/parameters.ts

```diff
@@ -39,9 +41,13 @@ export function transformParametersArray(
       if (paramComment) output += comment(paramComment);
 
       const required = paramObj.required ? `` : `?`;
-      output += `    "${paramName}"${required}: ${
-        paramObj.schema ? transformSchemaObj(paramObj.schema) : "unknown"
-      };\n`;
+      let type = ``;
```

Comment:
nit: can we use another name, other than `type`? Only because `type` is a special word in TypeScript (it doesn’t seem to be causing any problems here, but just for clarity)

Reply:
I see! I renamed it to `paramType`. https://github.com/drwpow/openapi-typescript/pull/489/commits/bb668eddc144b54ad422a2245eb54af4e2f63bf0